roofit/roofitcore/test/TestStatistics/testLikelihoodGradientJob.cpp

```diff
@@ -222,12 +231,15 @@ TEST_P(LikelihoodGradientJob, GaussianND)
    // --------
 
    RooFit::MultiProcess::Config::setDefaultNWorkers(NWorkers);
-   auto likelihood = std::make_shared<RooFit::TestStatistics::RooUnbinnedL>(pdf, data);
-   RooMinimizer m1(likelihood, RooFit::TestStatistics::LikelihoodMode::serial,
-                   RooFit::TestStatistics::LikelihoodGradientMode::multiprocess);
+   auto unbinned_l = std::make_shared<RooFit::TestStatistics::RooUnbinnedL>(pdf, data);
+   RooFit::TestStatistics::RooRealL likelihood("likelihood", "likelihood", unbinned_l);
+   RooMinimizer::Config cfg;
+   cfg.parallelLikelihood = false;
+   cfg.parallelGradient = true;
+   RooMinimizer m1(likelihood, cfg);
 
-   m1.setStrategy(0);
-   m1.setPrintLevel(-1);
+   RooMinimizer::setStrategy(0);
```

Comment:
Same story here. I have to explicitly set these here again for the test to succeed.